### ISSUE TITLE
What permission should the Cloudflare API have?

### ISSUE THREAD

**Issue by alexandrewillame** [QuicCloud/scripts#8]
It is mentioned in the Requirements to use with CloudFlare:

`A Cloudflare account with an API key.
`

What API Key exactly? Cloudflare requires to explicitly provide permission to the associated API, thus what permission should be given exactly so your script can work properly?

Thank you,

**Comment by alexandrewillame** [comment]
ok actually I found the problem, it was not intended to create a custom API Key, instead there is a general API Key for Cloudflare. I did a pull request to update the readme to include clearer information about where to find credentials:

To find those credentials:
* CF_EMAIL: simply your email for your account. Can be found on the top left corner of the Cloudflare website;
* CF_API_KEY: Go to your profile (top right corner icon) -> My Profile -> API Tokens -> Global API Key => View ;
* CF_ZONE_ID: Go to your home dashboard -> click on your website domain -> the Zine ID is at the bottom right side of the screen (see [this tutorial](https://developers.cloudflare.com/fundamentals/setup/find-account-and-zone-ids/)).

Closing this issue
